src/SourceBuild/patches/aspnetcore/0002-Silence-Ambiguous-FlushAsync.patch

```diff
@@ -0,0 +1,44 @@
+From: Larry Ewing <lewing@microsoft.com>
+
+Backport: https://github.com/dotnet/runtime/pull/85351
+
+diff --git a/src/Logging.AzureAppServices/src/BlobLoggerProvider.cs b/src/Logging.AzureAppServices/src/BlobLoggerProvider.cs
+index b85d798418..5fda9ca1b3 100644
+--- a/src/Logging.AzureAppServices/src/BlobLoggerProvider.cs
++++ b/src/Logging.AzureAppServices/src/BlobLoggerProvider.cs
+@@ -76,7 +76,9 @@ public class BlobLoggerProvider : BatchingLoggerProvider
+                     writer.Write(logEvent.Message);
+                 }
+ 
++#pragma warning disable CA2016
+                 await writer.FlushAsync().ConfigureAwait(false);
++#pragma warning restore CA2016
```

Comment:
Note this isn't how it is getting fixed in https://github.com/dotnet/aspnetcore/pull/47879. I'm not sure it matters or not.

Reply:
it will be removed as soon as aspnetcore makes it here so it doesn't matter

Reply:
Fix the link and merge on red.  The link needs to be right otherwise it is going to cause confusion for people working on source-build.